src/Command/SniffCommand.php

```diff
@@ -91,4 +91,21 @@ private function getFormatter(string $type, bool $pretty_format): FormatterInter
 
         return new ConsoleFormatter($pretty_format);
     }
+
+    private function guessStandard(?string $standard): string
+    {
+        if (null !== $standard) {
+            return $standard;
+        }
+
+        if (file_exists('csssniff.xml.dist')) {
+            return realpath('csssniff.xml.dist');
+        }
+
+        if (file_exists('csssniff.xml')) {
+            return realpath('csssniff.xml');
+        }
+
+        return 'Hostnet';
```

Comment:
Shall we simply provide a "Default" rather than "Hostnet"?

Reply:
I want keep the name Hostnet so I can show you are using the Hostnet CSS standard. If we rename it to "Default" if can be confusing what you are using.